ui/raidboss/data/03-hw/trial/sophia-ex.txt

```diff
@@ -14,7 +14,7 @@ hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
 
-0.0 "--sync--" sync /:Engage!/ window 0,1
+0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
 1.8 "--sync--" sync / 1[56]:[^:]*:Sophia:19C3:/ window 1.8,2
```

Comment:
I think this auto can be removed.

Reply:
Wasn't sure if this was an auto since it wasn't on the -ii list; but regardless, since there's a named ability at 12.0, makes sense.